src/wrapper/class_loader.rs

```diff
@@ -0,0 +1,59 @@
+use std::sync::Mutex;
+
+use errors::*;
+
+use JNIEnv;
+
+use objects::{GlobalRef, JObject, JClass, JValue};
+
+use strings::JNIString;
+
+use lazy_static::lazy_static;
+
+/// The `loadClass` function name.
+const LOAD_CLASS: &str = "loadClass";
+/// The `loadClass` signature.
+const LOAD_CLASS_SIG: &str = "(Ljava/lang/String;)Ljava/lang/Class;";
+
+lazy_static! {
+    /// The global class loader instance.
+    static ref CLASS_LOADER: Mutex<Option<GlobalRef>> = Mutex::default();
+}
+
+/// Register the given object as the global class loader instance.
+pub fn register_class_loader<'a>(env: &JNIEnv<'a>, class_loader: JObject<'a>) -> Result<()> {
+    // Check that the `loadClass` function is present.
+    env.get_method_id(class_loader, LOAD_CLASS, LOAD_CLASS_SIG)?;
```

Comment:
I'd suggest a little clearer way: `env.is_instance_of`.

However, it would also require a not null check :thinking: